neo4j-to-neptune/docs/example-conversion-config.yaml

```diff
@@ -0,0 +1,43 @@
+# Example label mapping and filtering configuration for Neo4j to Neptune conversion
+# This file demonstrates how to map vertex and edge labels and skip certain records during conversion
+
+# Vertex label mappings
+# Format: OldLabel: NewLabel
+vertexLabels:
```

Comment:
Part of me would prefer naming this something like `vertexLabelMappings` and `edgeLabelMappings`. The name `vertexLabels` on its own sounds to me like it is a list of labels to export. `vertexLabelMappings` is more explicit at the cost of being a bit longer. Thoughts?

Reply:
I'm slightly indifferent, only favouring the shorter version for less typing but open to renaming it.